openmc/tallies.py

```diff
@@ -1672,7 +1679,11 @@ def _align_tally_data(self, other):
             self_repeat_factor = other.num_nuclides
             other_tile_factor = self.num_nuclides
 
-            # Replicate the data
+            # Tile / repeat the tally data for the tally outer product
```

Comment:
Yeah, it can be tough to debug some of the tests. You might need to turn
off the hashing of inputs - i.e., just comment out the "cleanup" routine in
the TestingHarness class so that the inputs and outputs are left intact for
you to inspect and find the issue.
On Nov 19, 2015 6:07 PM, "sam" <notifications@github.com> wrote:

> In openmc/tallies.py
> <https://github.com/mit-crpg/openmc/pull/507#discussion_r45414857>:
>
> > @@ -1672,7 +1679,11 @@ def _align_tally_data(self, other):
> >              self_repeat_factor = other.num_nuclides
> >              other_tile_factor = self.num_nuclides
> >
> > -            # Replicate the data
> > +            # Tile / repeat the tally data for the tally outer product
>
> I'll look into the failed tests and see if I could find the issue.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-crpg/openmc/pull/507/files#r45414857>.
>